### PR TITLE
Feature/unit store testing

### PR DESCRIPTION
**Pull request by BrahimS**
@Zorin95670

### CODE REVIEW

File: frontend/src/composables/Products/useProductDetails.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ export default function useProductDetails() {
 			};
 		});
 		// productName.value = updatesData.name;
```
Comment (Zorin95670) [inline frontend/src/composables/Products/useProductDetails.js]: Remove this block of comment

File: frontend/changelog.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
+
+## [1.0.0] - 2022-05-04
```
Comment (Zorin95670) [inline frontend/changelog.md]: don't forget to improve version in packaqe.json and package-lock.json too

**Review by Zorin95670 — CHANGES_REQUESTED** [review]

File: frontend/changelog.md (hunk 2 above)
Comment (Zorin95670) [reply frontend/changelog.md]: version: [Unreleased] instead of 1.0.0

File: frontend/changelog.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+### Added
+
+- User store testing unit.
```
Comment (Zorin95670) [inline frontend/changelog.md]: Move this in changed

File: frontend/changelog.md (hunk 2 above)
Comment (BrahimS) [reply frontend/changelog.md]: it will be [Unreleased]

File: frontend/changelog.md (hunk 3 above)
Comment (BrahimS) [reply frontend/changelog.md]: ok

**Comment by BrahimS** [comment]
hello @Zorin95670 , voilà pour le changelog :)

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**